src/common/net/http-client.ts

```diff
@@ -7,6 +7,11 @@ import {
 } from '../interfaces/http-client.interface';
 
 export abstract class HttpClient implements HttpClientInterface {
+  readonly MAX_RETRY_ATTEMPTS = 3;
+  readonly BACKOFF_MULTIPLIER = 1.5;
+  readonly MINIMUM_SLEEP_TIME = 500;
```

Comment:
Nit: might be nice to denote as milliseconds.
```suggestion
  readonly MINIMUM_SLEEP_TIME_IN_MILLISECONDS = 500;
```